bench/wtperf/wtperf.c

```diff
@@ -753,14 +785,18 @@ run_mix_schedule(CONFIG *cfg, WORKLOAD *workp)
 	memset(workp->ops, WORKER_READ, sizeof(workp->ops));
 
 	pct = (workp->insert * 100) /
-	    (workp->insert + workp->read + workp->update);
+	    (workp->insert + workp->read + workp->truncate + workp->update);
```

Comment:
I have not currently excluded truncate from running alongside another workload.

If you think this limitation is worthwhile, I will add it.